src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -90,12 +90,17 @@ unsigned uct_rc_mlx5_iface_progress(void *arg)
 
 static ucs_status_t uct_rc_mlx5_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *iface_attr)
 {
-    uct_rc_iface_t *iface = ucs_derived_of(tl_iface, uct_rc_iface_t);
+    uct_rc_mlx5_iface_t *iface = ucs_derived_of(tl_iface, uct_rc_mlx5_iface_t);
+    uct_rc_iface_t *rc_iface   = &iface->super;
     ucs_status_t status;
 
-    status = uct_rc_iface_query(iface, iface_attr,
+    status = uct_rc_iface_query(rc_iface, iface_attr,
                                 UCT_IB_MLX5_PUT_MAX_SHORT(0),
-                                UCT_IB_MLX5_AM_MAX_SHORT(0),
+#if HAVE_IBV_EXP_DM
+                                iface->mlx5_common.dm.dm ? ucs_max(iface->mlx5_common.dm.dm->seg_len,
```

Comment:
would be nicer (from coding perspective) to use a local variable "max_inline" and pass it to uct_rc_iface_query() function